blog-posts/felix-interview.md

```diff
@@ -0,0 +1,65 @@
+---
+path: "/blog/interview-felix-hamann/"
+date: "2022-02-01"
+title: Felix Hamann – Senior Frontend Engineer 
+featuredImage: images/interview/felix-hero.jpg
+author: "Satellytes"
+authorSummary: "Interview mit Felix Hamann"
+teaserText: Felix Hamann ist der erste Angestellte von Satellytes und immer noch glücklich
+---
+
+
+## Interview mit Felix Hamann
+**This introduction text still to be done**
+
+<br>**Hallo Felix. Schön, dass Du Dir die Zeit nimmst. Stell Dich doch mal kurz vor.**<br>
+Hi! Ich bin Felix. Ich habe noch das Piepsgeräusch der 56k Modems im Ohr, bin also schon eine ganze Weile dabei. Seitdem ich denken kann faszinieren mich Computer und Webentwicklung.
+
+<br>**Du bist der erste Angestellte bei Satellytes. Bist Du nach all den Jahren immer noch happy? Highlights? Lowlights?**<br>
+Immer noch happy. Mit Ausrufezeichen! Highlights: Immer wieder ins kalte Wasser gestoßen zu werden, weil “Life begins at the end of your comfort zone”. Beispiele für "kaltes Wasser": Vor kurzem habe ich zum ersten Mal eine Präsentation auf einer Entwickler-Konferenz “emerge” gehalten und in einem meiner letzten Projekte hatte ich eine Managementrolle inne. Lowlights: Mir fällt eigentlich nur Corona ein, aber ich als introvertiertes Kellerkind bin zu Hause sowieso immer happy... trotzdem finde ich es schade, dass es letztes Weihnachten wieder keine solche legendäre Weihnachtsfeier wie 2019 auf der Hütte geben konnte.
```

Comment:
I think that's the correct spelling of eMerge.

https://globaldigitalfactory.allianz.com/blog/emerge-in-retrospect--.html